### PR TITLE
mynewt: wait for pairing in GAP/BOND/BON/BV-01-C

### PR DESCRIPTION
**Pull request by KKopyscinski**
give some time to finish pairing before disconnecting, to save the keys.

### CODE REVIEW

**Comment by michal-narajowski** [comment]
Does Mynewt have Security Level Changed event implemented? Or any other event that can be used to wait for pairing to finish?

**Comment by KKopyscinski** [comment]
@michal-narajowski I made this the way the other test is done, but yeah, maybe we can wait on encryption change

File: autopts/ptsprojects/mynewt/gap.py [hunk 1]
```diff
@@ -212,13 +212,13 @@ def test_cases(ptses):
...
-                  [TestFunc(btp.gap_set_io_cap, IOCap.no_input_output)],
+                  [TestFunc(btp.gap_set_io_cap, IOCap.no_input_output),
+                   TestFunc(btp.gap_wait_for_sec_lvl_change, 1, post_wid=108)],
                   generic_wid_hdl=gap_wid_hdl),
         ZTestCase("GAP", "GAP/BOND/BON/BV-03-C",
                   cmds=pre_conditions +
                   [TestFunc(btp.gap_set_io_cap, IOCap.no_input_output),
-                   # give some time before returning wid 118 to allow pair to finish
-                   TestFunc(sleep, 1, start_wid=118)],
+                   TestFunc(btp.gap_wait_for_sec_lvl_change, 1, post_wid=108)],
```
Comment (michal-narajowski) [inline autopts/ptsprojects/mynewt/gap.py:221]: Can we no add this wait in wid 118?

Comment (KKopyscinski) [reply autopts/ptsprojects/mynewt/gap.py:221]: So you only want to use this in GAP/BOND/BON/BV-01-C?

Comment (michal-narajowski) [reply autopts/ptsprojects/mynewt/gap.py:221]: No, I mean in wid/gap.py `hdl_wid_118()`.

Comment (KKopyscinski) [reply autopts/ptsprojects/mynewt/gap.py:221]: Ah, so in wid, not as post_wid. IIRC that wasn't working, but I'll take a look

Comment (KKopyscinski) [reply autopts/ptsprojects/mynewt/gap.py]: We can, I pushed the change, I think I was trying to wait at the end of 108, not in 118

**Review by michal-narajowski — APPROVED** [review]

**Pull request merged**